Mvc.NoFun.sln

```diff
@@ -78,6 +78,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Mvc.Ra
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.Mvc.RazorPages.Test", "test\Microsoft.AspNetCore.Mvc.RazorPages.Test\Microsoft.AspNetCore.Mvc.RazorPages.Test.csproj", "{0AB46520-F441-4E01-B444-08F4D23F8B1B}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "shared", "shared", "{704DD4E0-68D5-4768-9585-0F6D134B3350}"
+	ProjectSection(SolutionItems) = preProject
+		src\shared\PropertyValueSetter.cs = src\shared\PropertyValueSetter.cs
```

Comment:
The `PropertyHelper` type is internal to each of the two assemblies. I'll work around by passing in the setter delegate + the propertyinfo.